diff --git a/node_modules/@docusaurus/mdx-loader/lib/format.js b/node_modules/@docusaurus/mdx-loader/lib/format.js
index 61a54b6..8a0a63e 100644
--- a/node_modules/@docusaurus/mdx-loader/lib/format.js
+++ b/node_modules/@docusaurus/mdx-loader/lib/format.js
@@ -12,7 +12,6 @@ const path_1 = tslib_1.__importDefault(require("path"));
 // Copied from https://mdxjs.com/packages/mdx/#optionsmdextensions
 // Although we are likely to only use .md / .mdx anyway...
 const mdFormatExtensions = [
-    '.md',
     '.markdown',
     '.mdown',
     '.mkdn',
